src/main/java/net/fabricmc/loom/task/GenerateSourcesTask.java

```diff
@@ -90,26 +96,37 @@ public abstract class GenerateSourcesTask extends AbstractLoomTask {
 	private final DecompilerOptions decompilerOptions;
 
 	/**
-	 * The jar to decompile, can be the unpick jar.
+	 * The jar name to decompile, {@link MinecraftJar#getName()}.
 	 */
-	@InputFile
-	public abstract RegularFileProperty getInputJar();
-
-	/**
-	 * The jar used at runtime.
-	 */
-	@InputFile
-	public abstract RegularFileProperty getRuntimeJar();
```

Comment:
As tasks are part of the API (I think? They're very clearly and intentionally exposed to buildscripts), isn't this + adding a bunch of new required properties a breaking change?

Reply:
The Task API surface is not clearly defined (Something to possibly improve later), as far as I see it all of these properties past and present are not part of the public API, I doubt (hope) that no one was actually setting these.